llvm/lib/Target/SBF/Disassembler/SBFDisassembler.cpp

```diff
@@ -44,12 +44,7 @@ class SBFDisassembler : public MCDisassembler {
     SBF_ALU64 = 0x7
   };
 
-  enum SBF_SIZE {
-    SBF_W = 0x0,
-    SBF_H = 0x1,
-    SBF_B = 0x2,
-    SBF_DW = 0x3
-  };
+  enum SBF_SIZE { SBF_W = 0x0, SBF_H = 0x1, SBF_B = 0x2, SBF_DW = 0x3 };
```

Comment:
Ordinarily, re-formatting should only be done within the parts of the code changed in the patch itself. 
See https://llvm.org/docs/DeveloperPolicy.html: 
_Avoid committing formatting- or whitespace-only changes outside of code you plan to make subsequent changes to. Also, try to separate formatting or whitespace changes from functional changes, either by correcting the format first (ideally) or afterward. Such changes should be highly localized and the commit message should clearly state that the commit is not intended to change functionality, usually by stating it is [NFC](https://llvm.org/docs/Lexicon.html#nfc)._

This time, just go ahead and leave as is, but do become familiar with the above link and practices. Note that there are many formatting problems in the code that we inherited. They were originally left as is to facilitate upstream merges. That may be less important now.